modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartItemViews/ItemInfoView.js

```diff
@@ -11,23 +11,66 @@ import {sub} from 'frontend-js-web';
 import PropTypes from 'prop-types';
 import React, {useState} from 'react';
 
-function ItemInfoViewOptions({options}) {
-	return (
-		<div className="item-info-extra mt-3">
-			<h6 className="options">{options}</h6>
-		</div>
-	);
-}
+import {parseValue} from '../util/index';
 
-function ItemInfoViewBundle({childItems}) {
+function ItemInfoViewOptions({childItems, options}) {
 	const [expanded, setExpanded] = useState(false);
 
-	return Liferay.FeatureFlags['COMMERCE-9599'] ? (
+	const OptionsRenderer = (
+		<div className="child-items">
+			{options.map((option, index) => {
+				const {
+					skuId,
+					skuOptionName,
+					skuOptionValueNames,
+					value,
+				} = option;
+
+				const childItem = (childItems || []).find(
+					(childItem) => childItem.skuId === parseInt(skuId, 10)
+				);
+
+				const {name, quantity, skuUnitOfMeasure} = childItem || {};
+
+				return name ? (
+					<div className="item-info-extra pt-2" key={index}>
+						<h6 className="item-name">{skuOptionName}</h6>
+
+						<p className="item-sku">
+							<span>
+								<span>
+									{parseValue(skuOptionValueNames) ||
+										parseValue(value)}
+								</span>
+
+								<span className="pl-2">
```

Comment:
this span is what should be put in the if block, try to see if you can avoid code duplication